tests/v2/validation/snapshot/snapshot.go

```diff
@@ -35,40 +37,32 @@ const (
 	defaultNamespace             = "default"
 	DeploymentSteveType          = "apps.deployment"
 	isCattleLabeled              = true
-	IngressSteveType             = "networking.k8s.io.ingress"
+	initialIngress               = "ingress-before-restore"
+	initialWorkload              = "wload-before-restore"
+	ingressSteveType             = "networking.k8s.io.ingress"
 	ingressPath                  = "/index.html"
-	initialIngressName           = "ingress-before-restore"
-	initialWorkloadName          = "wload-before-restore"
-	localClusterName             = "local"
 	K3S                          = "k3s"
 	kubernetesVersion            = "kubernetesVersion"
 	namespace                    = "fleet-default"
 	port                         = "port"
+	postWorkload                 = "wload-after-backup"
 	ProvisioningSteveResouceType = "provisioning.cattle.io.cluster"
 	RKE1                         = "rke1"
 	RKE2                         = "rke2"
 	serviceAppendName            = "service-"
-	ServiceType                  = "service"
-	WorkloadNamePostBackup       = "wload-after-backup"
+	serviceType                  = "service"
 )
 
-type initialSnapshotConfig struct {
-	kubernetesVersion              string
-	initialControlPlaneUnavailable string
-	initialWorkerUnavailable       string
-	snapshot                       string
-}
-
 func snapshotRestore(t *testing.T, client *rancher.Client, clusterName string, etcdRestore *etcdsnapshot.Config) {
+	initialIngressName := namegen.AppendRandomString(initialIngress)
+	initialWorkloadName := namegen.AppendRandomString(initialWorkload)
+
```

Comment:
I'm just now noticing that we don't check for this existing after the snapshot is restored; we should also check that the postSnapshotWorkloads do not exist after the snapshot is restored. Can you make a ticket to track this?

Reply:
Discussed offline, resolved.